examples/offline_inference_vision_language.py

```diff
@@ -106,6 +106,19 @@ def run_minicpmv(question):
     return llm, prompt
 
 
+# InternVL
+def run_internvl(question):
+    # Generally, InternVL can use chatml template for conversation
+    TEMPLATE = "<|im_start|>User\n{prompt}<|im_end|>\n<|im_start|>Assistant\n"
+    prompt = f"<image>\n{question}\n"
+    prompt = TEMPLATE.format(prompt=prompt)
+    llm = LLM(
+        model="OpenGVLab/InternVL2-4B",
+        trust_remote_code=True,
```

Comment:
Can you add a `max_num_seqs=5` here similarly to what we did with `Phi3-Vision`? This example wouldn't run for me otherwise on a A100-80G.

Reply:
Updated